### ISSUE TITLE
Sort: missing strong visible indication on focus (2.4.7)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#935]
### Expected behavior
Upon focus, the element displays the SKY UX visual blue gradient border.

### Actual behavior
The element's grey border darkens slightly which difficult to notice.

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.11.0](https://github.com/blackbaud/skyux/releases/tag/v1.11.0)
